test/automated/index.js

```diff
@@ -0,0 +1,211 @@
+/* eslint-env node, mocha, es6 */
+
+'use strict';
+
+const { before, after, beforeEach, describe, it } = require('mocha');
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+
+chai.use(chaiAsPromised);
+const { expect } = chai;
+
+require('chromedriver');
+const selenium = require('selenium-webdriver');
+const until = require('selenium-webdriver/lib/until');
+
+describe('Likely', function () {
+    let driver;
+
+    before(function () {
+        driver = new selenium.Builder()
+            .forBrowser('chrome')
+            .build();
+    });
+
+    after(function () {
+        return driver.quit();
+    });
+
+    describe('common operations', function () {
+        const mockedCounterValue = '10';
+
+        before(function () {
+            // The browser could start long
+            this.timeout(20000);
+
+            return getLikely(driver, 'http://ilyabirman.github.io/Likely/autoinit.html', { waitUntilInitialized: true })
+        });
+
+        it('should fetch the counters for Facebook', function () {
+            return expectToContainText(driver, '.likely__counter_facebook', mockedCounterValue);
+        });
+
+        it('should fetch the counters for Google+', function () {
+            return expectToContainText(driver, '.likely__counter_gplus', mockedCounterValue);
+        });
+
+        it('should fetch the counters for Odnoklassniki', function () {
+            return expectToContainText(driver, '.likely__counter_odnoklassniki', mockedCounterValue);
+        });
+
+        it('should fetch the counters for Pinterest', function () {
+            return expectToContainText(driver, '.likely__counter_pinterest', mockedCounterValue);
+        });
+
+        it('should fetch the counters for VK', function () {
+            return expectToContainText(driver, '.likely__counter_vkontakte', mockedCounterValue);
+        });
+
+        it('should open the sharing dialog for Facebook', function () {
+            this.timeout(10000);
+            return expectClickToOpen(driver, '.likely__widget_facebook', /facebook\.com/);
+        });
+
+        it('should open the sharing dialog for Google+', function () {
+            this.timeout(10000);
```

Comment:
also seems it could be automated a little bit:

```js
[{ id: 'vk', urlReg: /vk\.com/ }, { ... }, ...].forEach(({ id, urlReg }) => {
  it(id, function () {
    return expectClickToOpen(driver, `.likely__widget_${id}`, urlReg);
  })
})
```